src/components/policy/policy_regular/include/policy/status.h

```diff
@@ -117,6 +118,39 @@ class Status {
   const PolicyTableStatus enum_status_;
 };
 
+/**
+ * @brief The UpdatePendingStatus class represents cases when SDL knows that an
+ * update is required and but before the snapshot is sent to the HMI
```

Comment:
@mked-luxoft redundant `and`

Reply:
removed in d35964dff07fd3af91d33ba543064bfddabb1446